schema-dex-amm-extended.graphql

```diff
@@ -214,11 +226,23 @@ type DexAmmProtocol implements Protocol @entity {
   " Current TVL (Total Value Locked) of the entire protocol "
   totalValueLockedUSD: BigDecimal!
 
+  " The sum of all active and non-active liquidity in USD for this pool. "
+  totalLiquidityUSD: BigDecimal!
+
+  " All liquidity in USD that is active. Will be equal to totalLiquidity except for in concentrated liquidity - where activeLiquidity is all liquidity positions that contain the pools current tick. "
+  activeLiquidityUSD: BigDecimal!
```

Comment:
Well - its a strange one, because total and active liquidity will always be the same for non-concentrated liquidity - at least what I have seen. Though the concepts are not as useful, it is technically true. Though, it does not add information if you are aware of how the protocol works.